examples/community/lpw_stable_diffusion.py

```diff
@@ -268,6 +292,16 @@ def get_weighted_text_embeddings(
         skip_weighting (`bool`, *optional*, defaults to `False`):
             Skip the weighting. When the parsing is skipped, it is forced True.
     """
+    # set lora scale so that monkey patched LoRA
+    # function of text encoder can correctly access it
+    if lora_scale is not None and isinstance(pipe, LoraLoaderMixin):
```

Comment:
We recently shipped https://github.com/huggingface/diffusers/pull/8981. I think this should be StableDiffusionLoraLoaderMixin. Maybe @sayakpaul can provide more clarity

Reply:
`LoraLoaderMixin` is fine but it's just deprecated. Prefer using `StableDiffusionLoraLoaderMixin` as @a-r-r-o-w mentioned.